### PR TITLE
string-functions: add instructions for using CONCAT() and CONCAT_WS()

### PR DESCRIPTION
**Pull request by Weaxs**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

fix https://github.com/pingcap/docs/issues/15484

This PR adds instructions for the string functions CONCAT() and CONCAT_WS().

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by xzhangxian1008 — APPROVED** [review]
LGTM

**Comment by Weaxs** [comment]
> Hi @Weaxs, I've submitted the commit [bec7a18](https://github.com/pingcap/docs-cn/commit/bec7a1829bb388a106abd2a64cb77a2ef01a6aaf) to refine some descriptions and unify the formats. Would you please check if the commit is OK? Thanks.

I've checked the commit and it looks OK

**Review by qiancai — APPROVED** [review]

File: functions-and-operators/string-functions.md [hunk 1]
```diff
@@ -100,11 +100,184 @@ SELECT BIN(-7);
...
+| Ti          |
++-------------+
+| TiDB Server |
++-------------+
+```
 
 ### [`CONCAT_WS()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_concat-ws)
 
-返回由分隔符连接的字符串
+`CONCAT_WS()` 函数是一种带分隔符的 `CONCAT()`，返回由分隔符连接的字符串。
```
Comment (qiancai) [inline functions-and-operators/string-functions.md:165]:
```suggestion
`CONCAT_WS()` 函数是一种带分隔符的 [`CONCAT()`](#concat)，返回由分隔符连接的字符串。
```

**Pull request merged**